apps/laboratory/docker-canary.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/env/bin
+
+# Not adding `set -e` so that S3 upload happens regardless
+
+npm run playwright:canary:test
+
+destination="s3://$TEST_RESULTS_BUCKET/web3modal-canary/$(date --iso-8601=seconds)/test-results/"
+echo "Uploading test results to $destination"
+aws s3 cp ./test-results/ $destination --recursive
```

Comment:
minor nitpick: dis fails now if the container doesn't have AWS credis which breaks some Developer Experience use-cases

Reply:
What use case does it break?

I had a comment here explaining why I had to do it this way: https://github.com/WalletConnect/web3modal/pull/2169#discussion_r1572704269

Reply:
We can make it optional e.g. with an env var in another PR if we need it